hackathon_site/dashboard/frontend/src/pages/AdminDashboard/AdminDashboard.tsx

```diff
@@ -2,12 +2,85 @@ import React from "react";
 import Typography from "@material-ui/core/Typography";
 import Header from "components/general/Header/Header";
 import { hackathonName } from "constants.js";
+import Grid from "@material-ui/core/Grid";
+import ImageList from "@material-ui/core/ImageList";
+import ImageListItem from "@material-ui/core/ImageListItem";
+import styles from "./AdminDashboard.module.scss";
+import { mockPendingOrders, overviewTitles } from "testing/mockData";
+import Paper from "@material-ui/core/Paper";
+import Card from "@material-ui/core/Card";
+import OrderCard from "components/orders/OrderCard/OrderCard";
+import Button from "@material-ui/core/Button";
+import { makeStyles } from "@material-ui/core/styles";
 
 const AdminDashboard = () => {
+    const useStyles = makeStyles({
+        imageList: {
+            flexWrap: "nowrap",
+            transform: "translateZ(0)",
+        },
+    });
+    const classes = useStyles();
     return (
         <>
             <Header />
             <Typography variant="h1">{hackathonName} Admin Dashboard</Typography>
+            <Grid className={styles.dashboard}>
```

Comment:
![image](https://user-images.githubusercontent.com/20618460/211699777-08d63faa-00a0-4036-86a5-962dbd6dc57f.png)
The spacing on the overview cards is very wide on large screens, i think a fixed size spacing is better (similar to the order cards). Also the overview cards will have a fixed number (refer to the figma) so there will be at most 5 cards which stack on smaller screens (again, refer to the figma https://www.figma.com/file/1bYQy6ioaZwxPD9w6D47jW/Hardware-signout-site?node-id=2640%3A23301&t=dQj2pNeHVtBsZp9c-0)